src/test/java/co/rsk/bitcoinj/script/FlyoverRedeemScriptParserTest.java

```diff
@@ -22,9 +22,9 @@
 
 public class FlyoverRedeemScriptParserTest {
 
-    private final List<BtcECKey> defaultRedeemScriptKeys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
-    private final List<BtcECKey> emergencyRedeemScriptKeys = RedeemScriptUtils.getEmergencyRedeemScriptKeys();
-    private final Sha256Hash derivationArgumentsHash = Sha256Hash.of(new byte[]{1});
+    private final List<BtcECKey> keys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
```

Comment:
Could you please use a meaningful variable name? WDYT?

Reply:
Any suggestion?